docs/Build-for-RPi2.md

```diff
@@ -3,10 +3,11 @@
 ### Prerequisite
 #### Linux
 
-Install arm linux cross compiler.
+Install arm linux cross compiler and i2c library.
 
 ``` bash
-sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf
+sudo apt-get install gcc-arm-linux-gnueabihf g++-arm-linux-gnueabihf i2c-tools libi2c-dev
+sudo mv /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h /usr/arm-linux-gnueabihf/include/linux/i2c-dev.h.kernel
```

Comment:
'libi2c-dev' package is mandatory. It include 'i2c-dev.h' file which has same name in Linux kernel. That's why renaming is needed. It rename original 'i2c-dev.h' to 'i2c-dev.h.kernel' in host file system, but not in arm include folder. That's why it is needed to change file name manually. If you have any other solution, please let me know.

I think 'i2c-tools' is not mandatory. I'll remove it.